test/acl.test.js

```diff
@@ -358,39 +358,39 @@ describe('security ACLs', function() {
           log('Role: ', myRole.toObject());
 
           myRole.principals.create({principalType: RoleMapping.USER, principalId: userId},
-          function(err, p) {
-            log('Principal added to role: ', p.toObject());
-
-            ACL.create({
-              principalType: ACL.ROLE, principalId: 'MyRole',
-              model: 'Customer', property: ACL.ALL,
-              accessType: ACL.READ, permission: ACL.DENY,
-            }, function(err, acl) {
-              log('ACL 2: ', acl.toObject());
-
-              ACL.checkAccessForContext({
-                principals: [
+            function(err, p) {
+              log('Principal added to role: ', p.toObject());
```

Comment:
The changes in this file look like purely whitespace-related, could you please revert?